src/libs/ReportSecondaryActionUtils.ts

```diff
@@ -604,6 +595,58 @@ function isMergeAction(parentReport: Report, reportTransactions: Transaction[],
     return isMoneyRequestReportEligibleForMerge(parentReport.reportID, isAdmin);
 }
 
+function isMergeActionForSelectedTransactions(transactions: Transaction[], reports: Report[], policies: Policy[]) {
+    if ([transactions, reports, policies].some((collection) => collection?.length > 2)) {
+        return false;
+    }
+
+    if (transactions.length === 2) {
+        const currentUserAccountID = getCurrentUserAccountID();
```

Comment:
Just wanna confirm that we can't use conditions of `isMoneyRequestReportEligibleForMerge` and `areTransactionsEligibleForMerge` for those checks, can we?

Reply:
No those checks are different. This check is only needed from the Reports page too.